source/vibe/db/postgresql/package.d

```diff
@@ -194,8 +195,9 @@ class Dpq2Connection : dpq2.Connection
 
     private void waitEndOfReadAndConsume(in Duration timeout)
     {
-        auto event = waitEndOfRead(timeout);
-        scope(exit) destroy(event); // Prevents 100% CPU usage
+        auto event = socketEvent();
```

Comment:
This function in fact now does `waitAnyEventAndConsume`.
Will it be called not too often for events that do not imply arrival of new data?